release-notes/topics/aap-212.adoc

```diff
@@ -0,0 +1,56 @@
+* Updated the collection counts on the *Content* tab when the collection details were filtered
+ 
+.Bug fixes
+* Fixed an issue where Controller user interface would not load on a few cloud providers
+* Added compatibility with earlier versions for `image_pull_secret` key in `awx-operator`, so that the backup and restore operations are completed successfully
+* Automation controller backups no longer fail when multiple secrets are supported
+* Resolved certification mismatch during upgrade so that every controller dashboard is accessible
+* Improved the labels for Japanese translations
+* Resolved issues when upgrading automation hub from version 2.1.1 to version 2.1.2
+* Resolved issues where the Controller user interface was not accessible, when automation controller was installed through Operator on OpenShift Core Platform 4.10
+* Fixed an issue that caused the upgrade from Ansible Automation Platform 1.2 to 2.1 to fail, when the execution environment was transferred to automation hub
+* Fixed an issue so that a standalone installation of private automation hub can now transfer the execution environments to the hub
+* Fixed an issue when the variable `sslmode` was not configured correctly during automation hub installation
+* Running the ``./setup.sh -b` command from the installer directory no longer fails to load group variables
+* Fixed an issue where the SELinux context was not being configured correctly on subsequent installations
+* The installer no longer fails when IPV6 is disabled
+* Modified the database backup and restore logic to compress the data dump
+* Fixed an issue during installation where a configured proxy would not be used when required
+* Fixed installations of execution environments when installing without internet access
+* Creating default execution environments no longer fails when the password includes special characters
+* Added support to set a parameter to change the temporary directory path, so that the execution environments can be extracted to the bundled installer
+* Upgrading to Ansible Automation Platform 2.1 no longer fails when the Django superuser is missing
+* Fixed an issue where installation of automation hub failed due to symlinks
+* Added explanations for variables of service catalog worker in the installer inventory
+* Fixed an issue where installing automation hub with an external database did not install Postgres as expected
+* Fixed an issue where variables with default values were shown as undefined variables
+* Added support for `ansible_host` in the installer inventory for automation hub
+* Fixed an issue where a few commands were missing on initializing the container
+* Fixed an issue where the TTL controller caused the jobs to run repeatedly
+* Resolved intermittent `500 internal server error` while pulling the execution environments from private automation hub
```

Comment:
Our changelog (https://github.com/ansible/galaxy_ng/pull/1178/files) for 2.1.2/4.4.3 also mentions fixing the 500 server error when pushing EE's. Should this also be mentioned here?

Reply:
I modified the language of this note to reflect this.